locksmith/src/routes/v2/subscriptions.ts

```diff
@@ -13,6 +14,7 @@ router.get(
   '/:network/locks/:lockAddress/keys/:keyId',
```

Comment:
Is this endpoint used by verifiers when the scan tickets?

Reply:
That's a good question. I don't think we use it in the verifiers since that's only for ticket which doesn't require subscription.

Reply:
We don't.